README.md

```diff
@@ -11,9 +11,9 @@
 [![Jupyter book](https://jupyterbook.org/badge.svg)](https://bandframework.github.io/Bfrescox)
 
 
-__PROVIDE FRESCOX INFORMATION AND LINKS__
+Bfrescox is a Python package wrapping [frescox](https://github.com/LLNL/Frescox), a Fortran library for coupled-reaction-channels calculations in nuclear physics.  The intention of this package is to provide a user-friendly experience to perform parametric reaction calculations and uncertainty quantification studies with frescox.
```

Comment:
> a Fortran library

application, program, software package, software tool?

In official documentation, I see `Fresco` and `Frescox` with capital "F".

> "studies with Frescox"

studies with a Frescox executable that is built from the code in the Frescox repository and installed automatically in Bfrescox during Bfrescox installation.

Specify citation?  I have seen [others request](https://github.com/bandframework/surmise) citation of User Guide.